content/cookbook/programs/change-account-size.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to change account size
+sidebarSortOrder: 3
+description: "Learn how to change the size of an account in a Solana program."
+---
+
+You can change a program owned account's size with the use of `realloc`.
+`realloc` can resize an account up to 10KB. When you use `realloc` to increase
+the size of an account, you must transfer lamports in order to keep that account
+rent-exempt.
+
+```rust filename="realloc.rs"
+use {
+  crate::{
+      instruction::WhitelistInstruction,
+      state::WhiteListData,
+  },
+  borsh::{BorshDeserialize, BorshSerialize},
+  solana_program::{
+      account_info::{next_account_info, AccountInfo},
+      entrypoint::ProgramResult,
+      msg,
+      program::invoke_signed,
+      program::invoke,
+      program_error::ProgramError,
+      pubkey::Pubkey,
+      sysvar::Sysvar,
+      sysvar::rent::Rent,
+      system_instruction,
+  },
+  std::convert::TryInto,
+};
```

Comment:
Rust is formatted in 4 spaces per `rustfmt`.

You can fix these easily: I just paste into a `deleteme.rs` file, save and let vscode format, then paste back.